### DISCUSSION TITLE
2.10.2

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#3694] [Design & tech]
## What's Changed
* Fix return type in `instantFromLocalDate` type def by @innuendonh in https://github.com/elsa-workflows/elsa-core/pull/3618
* Initialize tenantId in specification constructors. by @AlesDo in https://github.com/elsa-workflows/elsa-core/pull/3657
* Activities Kafka: Changed LogLevel to Information by @Snotax in https://github.com/elsa-workflows/elsa-core/pull/3663
* Activities Kafka: Implemented AutoOffsetReset Property in server and activity by @Snotax in https://github.com/elsa-workflows/elsa-core/pull/3665
* feat: 🎸 AzureServiceBus - TenantIdResolver by @Nico-VanHaaster in https://github.com/elsa-workflows/elsa-core/pull/3668
* WorkflowDefinitions: OrderBy and SortBy by @brunomartinspro in https://github.com/elsa-workflows/elsa-core/pull/3662
* fix: add missing tenant id specifiers; limit tailwind css to elsa-studio-root by @sjd2021 in https://github.com/elsa-workflows/elsa-core/pull/3673
* Added missing params to WorkflowDashboard component, limited model change handler by @sabitertan in https://github.com/elsa-workflows/elsa-core/pull/3676

## New Contributors
* @innuendonh made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/3618
* @AlesDo made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/3657
* @Snotax made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/3663
* @sjd2021 made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/3673

**Full Changelog**: https://github.com/elsa-workflows/elsa-core/compare/2.10.0...2.10.2

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-core/releases/tag/2.10.2'>2.10.2</a>.</em>

**Comment by nickbeau** [comment]
Hi All. Looks like Elsa.Designer.Components.Web is broken in this release. See #3695
